README.md

```diff
@@ -7,17 +8,22 @@
 https://marketplace.visualstudio.com/items?itemName=0x9ef.vscode-vlang&ssr=false
 
 ## Features
-* Code snippets
-* Syntax highlighting 
+
+* Snippets
+* Syntax highlighting
+* Code formatting
```

Comment:
This is the main addition, the rest is mostly spacing recommended by my markdown formatter